modules/pensions/app/sidekiq/pensions/pension_benefit_intake_job.rb

```diff
@@ -96,6 +98,20 @@ def init(saved_claim_id, user_account_uuid)
       @intake_service = BenefitsIntake::Service.new
     end
 
+    ##
+    # Check FormSubmissionAttempts for record with 'pending' or 'success'
+    #
+    # @return false if unable to find a FormSubmission, matching the intake.uuid
+    # or there are no matching FormSubmissionAttempt aasm_states
+    #
+    def form_submission_pending_or_success
+      form_submission = @claim&.form_submissions&.find do |submission|
+        submission.benefits_intake_uuid == @intake_service.uuid
```

Comment:
Will the UUID be the same on retry, since the intake service is new each time?

Reply:
I was curious about that. The retry would hit the 'init' each tile and generate a new uuid? I'll make the adjustment to just check all submissions attached to the saved_claim_id

Reply:
I believe so, yes